validator/webui/index.html

```diff
@@ -456,6 +456,11 @@
           button.addEventListener('click', function() {
             var urlToFetch = input.value.trim();
             if (urlToFetch.length > 0) {
+              if (!(urlToFetch.indexOf('http://') == 0 ||
+                    urlToFetch.indexOf('https://') == 0)) {
+                urlToFetch = 'https://' + urlToFetch;
```

Comment:
I don't think just using https is going to work here as not all hosts support https. Should we try with https and if get nothing back then try again with http?